run_local_with_dependencies.sh

```diff
@@ -2,6 +2,6 @@
 
 sm2 -start MONGO DATASTREAM EMAIL HMRC_EMAIL_RENDERER MAILGUN_STUB DESKPRO_STUB DESKPRO_TICKET_QUEUE
 
-sm2 -start THIRD_PARTY_APPLICATION THIRD_PARTY_DEVELOPER API_SUBSCRIPTION_FIELDS API_DEFINITION THIRD_PARTY_DELEGATED_AUTHORITY API_PLATFORM_EVENTS API_PLATFORM_MICROSERVICE THIRD_PARTY_ORCHESTRATOR PUSH_PULL_NOTIFICATIONS_API API_PLATFORM_XML_SERVICES PUSH_PULL_NOTIFICATIONS_GATEWAY
+sm2 -start THIRD_PARTY_APPLICATION THIRD_PARTY_DEVELOPER API_SUBSCRIPTION_FIELDS API_DEFINITION THIRD_PARTY_DELEGATED_AUTHORITY API_PLATFORM_EVENTS API_PLATFORM_MICROSERVICE THIRD_PARTY_ORCHESTRATOR # PUSH_PULL_NOTIFICATIONS_API API_PLATFORM_XML_SERVICES PUSH_PULL_NOTIFICATIONS_GATEWAY
```

Comment:
Could split this into two lines, so that the comment is easy to see on narrow screens
```suggestion
sm2 -start THIRD_PARTY_APPLICATION THIRD_PARTY_DEVELOPER API_SUBSCRIPTION_FIELDS API_DEFINITION THIRD_PARTY_DELEGATED_AUTHORITY API_PLATFORM_EVENTS API_PLATFORM_MICROSERVICE THIRD_PARTY_ORCHESTRATOR
#sm2 -start PUSH_PULL_NOTIFICATIONS_API API_PLATFORM_XML_SERVICES PUSH_PULL_NOTIFICATIONS_GATEWAY
```

Reply:
No - this is a bug in waiting - I'd commented out stuff cos it was blowing up my memory...  I need to uncomment for those with 128GB of memory...